vernac/declareUniv.ml

```diff
@@ -33,6 +33,16 @@ type universe_name_decl = {
   udecl_anon : UGlobal.t list;
 }
 
+type [@warning "-37"] sort_source =
+  | BoundQuality 
```

Comment:
Do you intend to add section level sort variables? It seems this code on
~~~
Set Universe Polymorphism.
Section S.
Sort s.
Definition foo := Type@{s|Set}.
End S.
~~~
will produce a global sort `s` and effectively monomorphic `foo`.